### PR TITLE
Tweaked logo dimensions and style adjustments for header and featured item section

### PR DESCRIPTION
**Pull request by ken-ellingsen**
Started with some changes to the Featured Item section and ended up tweaking the Header a bit. I also edited the logo so the dimensions were easier to work with for styling.

### CODE REVIEW

File: src/components/HeaderNav.jsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const HeaderNav = () => {
                 <li><Link to="#" className="header--nav__link">About</Link></li>
                 <li><Link to="#" className="header--nav__link">Contact</Link></li>
                 <li><Link to="#" className="header--nav__link">Add Credits</Link></li>
-                <li><Link to="/Cart" className="header--nav__link">Cart | 3</Link></li>
+                <li><Link to="/Cart" className="header--nav__link"><span>Cart | 3</span></Link></li>
```
Comment (michaella23) [inline src/components/HeaderNav.jsx:13]: I think maybe the `span` would just be on the quantity, not the 'Cart |' text, right?

**Review by michaella23 — COMMENTED** [review]
Thanks for doing this image fix. That thing was unwieldy!!

File: src/components/HeaderNav.jsx (hunk 1 above)
Comment (ken-ellingsen) [reply src/components/HeaderNav.jsx:13]: I added it to whole `li` to make the the whole cart link look like a button, can definitely just do the number if we want though. Other option could be to have two `span`s and have the second also highlight the number in the cart "button".

Comment (michaella23) [reply src/components/HeaderNav.jsx:13]: Ah, I was just thinking about that number changing dynamically. We'll just leave it as is and can sort it out as we go.

**Pull request merged**